src/UID2.Client/UID2Encryption.cs

```diff
@@ -39,23 +41,36 @@ internal static DecryptionResponse Decrypt(string token, KeyContainer keys, Date
             {
                 return DecryptV2(Convert.FromBase64String(token), keys, now, domainName, clientType);
             }
-
-            if (data[1] == (int)AdvertisingTokenVersion.V3)
+            else
             {
-                return DecryptV3(Convert.FromBase64String(token), keys, now, identityScope, 3, domainName, clientType);
-            }
+                // Assume that newer tokens have an identity scope and type prefix
```

Comment:
Can we also comment the impact of a new token not having these things at byte 0? Or having new scopes/types that we don't know about?

Reply:
Done